imagepicker/src/main/java/com/esafirm/imagepicker/features/common/BaseConfig.kt

```diff
@@ -7,4 +7,5 @@ abstract class BaseConfig {
     abstract var savePath: ImagePickerSavePath
     abstract var returnMode: ReturnMode
     abstract var isSaveImage: Boolean
+    abstract var showDoneButtonAlways: Boolean
```

Comment:
I think you should only add this in `ImagePickerConfig` since it's not affecting `CameraOnly` functionality?